extensions/sql/asset/sql_asset_index_loader_proposal.md

```diff
@@ -0,0 +1,186 @@
+## ER Diagramm
+
+![ER Diagram](http://www.plantuml.com/plantuml/png/XP513e8m44NtFSMiJNe1GWWhDnwY6Ud4b40XdS58XBjRnI346dVD_DVVtpyb2mOPsaQH5oSZaAqCCfF0NG4Su7KspcR04fo_nA7MQbPVlB4eYDO6Olvn5_ByU2gAnU89zA7hAOWZpa3e9X6ekVCHIJtmfiU_xglnQ3osj8yE7t4PcoismmfjzUvSKoLxVHznBMkQj4vtiO7c22N-eqhoDgxW0G00)
+
+## Table schema
+
+```sql
+
+-- table: edc_asset
+CREATE TABLE IF NOT EXISTS edc_asset (
+    asset_id VARCHAR(255) NOT NULL,
+    PRIMARY KEY (asset_id)
+);
+
+
+-- table: edc_asset_dataaddress
+CREATE TABLE IF NOT EXISTS edc_asset_dataaddress (
+    asset_id VARCHAR(255) NOT NULL,
+    properties VARCHAR(MAX) NOT NULL,
+    PRIMARY KEY (asset_id),
+    FOREIGN KEY (asset_id) REFERENCES edc_asset (asset_id) ON DELETE CASCADE
+);
+
+COMMENT ON COLUMN edc_asset_dataaddress.properties is 'DataAddress properties serialized as JSON';
+
+
+-- table: edc_asset_property
+CREATE TABLE IF NOT EXISTS edc_asset_property (
+    asset_id VARCHAR(255) NOT NULL,
+    property_name VARCHAR(255) NOT NULL,
+    property_string_value VARCHAR(MAX) DEFAULT NULL,
+    property_object_value VARCHAR(MAX) DEFAULT NULL,
```

Comment:
I guess `property_string_value` and `property_object_value` could be collated into one, given that we'll have a `property_type` field containing the java class name